src/ts/utils/random-id-generator.ts

```diff
@@ -0,0 +1,30 @@
+export const FEED_SHOWN_ID_LENGTH = 12;
```

Comment:
instead of a new folder, we should update the existing file - switching the suffix to `ts` and updating the webpack/babel config to recognize both types of files as worthy of bundling.

Reply:
oh looks like the config has already been updated! then just the file extension swap _should_ put us in a situation to test the full build of the player